app/javascript/components/patient/assessment/actions/CurrentStatus.js

```diff
@@ -12,43 +12,43 @@ class CurrentStatus extends React.Component {
   generateStatus(status) {
     if (status === 'exposure_symptomatic') {
       return (
-        <Badge variant="danger" className="badge-larger-font" aria-label="Notification Status Exposure Symptomatic">
+        <Badge variant="danger" className="badge-larger-font" aria-label="Linelist: Symptomatic.">
```

Comment:
These aria labels made no sense before and didnt really work. If you want to test how bad they were, try to use a screen-reader on the Patient Detail page on Demo. It's so bad. This clears it up.

Reply:
should we keep the workflow information in here? something like Workflow: Exposure, Linelist: Symptomatic or something?